doc/fr/Sil/Component/Invoice/domain/invoice.rst

```diff
@@ -0,0 +1,53 @@
+Invoice
+-------
+
+Les factures sont représentées par le modèle Invoice.
+
+La propriété ``createdAt`` prendra sa valeur lors du changemenent du statut de la facture à "validé".
```

Comment:
May be a field `validatedAt` should be used to « log » the date when validated state is set ?